react/features/toolbar/components/Toolbar.native.js

```diff
@@ -137,18 +138,36 @@ class Toolbar extends AbstractToolbar {
                     onClick = { this._onRoomLock }
                     style = { style }
                     underlayColor = { underlayColor } />
+                <ToolbarButton
+                    iconName = 'star'
```

Comment:
@yanas I picked a random icon here because I didn't know how to express "toggle audio only mode" suggestions are welcome :-)